hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/cli/GenericCli.java

```diff
@@ -52,6 +53,10 @@ public class GenericCli implements Callable<Void>, GenericParentCommand {
 
   public GenericCli() {
     cmd = new CommandLine(this);
+    cmd.setExecutionExceptionHandler((ex, commandLine, parseResult) -> {
+      printError(ex);
```

Comment:
Thanks @duongnguyen0 for the contribution. Overall patch looks nice.

Existing  [GenericCli.java#L88](https://github.com/apache/ozone/blob/master/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/cli/GenericCli.java#L88) `printError()` function is using the exception cause, if its not null. Could you please check the print message in your patch and whether its printing all the necessary info.
`printError(ex.getCause() == null ? ex : ex.getCause());`

Reply:
Thanks for the thorough review @rakeshadr. 
Indeed, the extraction of the ExecutionException's cause happens in the caller of ExecutionExceptionHandler, aka. `CommandLine.execute`.

```
    public int execute(String... args) {
        
        try {
            parseResult[0] = parseArgs(args);
            return enrichForBackwardsCompatibility(getExecutionStrategy()).execute(parseResult[0]);
        } catch (ParameterException ex) {
            ...
        } catch (ExecutionException ex) {
            try {
                Exception cause = ex.getCause() instanceof Exception ? (Exception) ex.getCause() : ex;
                return getExecutionExceptionHandler().handleExecutionException(cause, ex.getCommandLine(), parseResult[0]);
            } ....
    }
```

Basically, the same printed error message is maintained. And I did a comparison to verify it and put a sample in the PR's description.

Reply:
Thanks @duongnguyen0 for the clear explanation.